tensorflow/stream_executor/dnn.cc

```diff
@@ -122,7 +122,11 @@ string PadAlignmentString(PadAlignment alignment) {
       return "cuDNN padding";
     case PadAlignment::kTensorFlowPadding:
       return "TensorFlow padding";
+    default:
+      LOG(FATAL) << "Unknown padding alignment "
```

Comment:
nit: maybe just fall through instead of log fatal (e.g., in stream.cc change below, you don't LOG(FATAL) in the default case).  Having a default: case will make it so that a new PadAlignment value won't trigger a warning here for not handling all cases...